047-TrafficControlWithDapr/Student/Challenge-00.md

```diff
@@ -0,0 +1,323 @@
+    az deployment sub create --location "<resource-group-location>" --template-file rg.bicep --parameters ./main.parameters.json --query "properties.outputs" --output yamlc
+    ```
+
+1.  Upon completion, the command will display the name of the newly-created resource group:
+
+    ```yaml
+    resourceGroupName:
+    type: String
+    value: rg-dapr-youruniqueid123
+    ```
+
+    Copy the resource group as you'll use in the next command.
+
+1.  You'll now create the required Azure resources inside the new resource group with the following Azure CLI command.
+
+    ```shell
+    az deployment group create --resource-group "rg-dapr-youruniqueid123" --template-file main.bicep --parameters ./main.parameters.json --query "properties.outputs" --output yaml
+    ```
+
+    *Creating the resources can take some time. You're encouraged to jump to the **challenge 01** while the command executes.*
+
+    Upon completion, the command will output information about the newly-created Azure resources:
+
+    ```yaml
+    aksFQDN:
+      type: String
+      value: dapr-mce123-609718f5.hcp.southcentralus.azmk8s.io
+    aksName:
+      type: String
+      value: aks-dapr-mce123
+    aksazurePortalFQDN:
+      type: String
+      value: dapr-mce123-609718f5.portal.hcp.southcentralus.azmk8s.io
+    containerRegistryLoginServerName:
+      type: String
+      value: crdaprmce123.azurecr.io
+    containerRegistryName:
+      type: String
+      value: crdaprmce123
+    eventHubEntryCamName:
+      type: String
+      value: ehn-dapr-mce123-trafficcontrol/entrycam
+    eventHubExitCamName:
+      type: String
+      value: ehn-dapr-mce123-trafficcontrol/exitcam
+    eventHubNamespaceHostName:
+      type: String
+      value: https://ehn-dapr-mce123-trafficcontrol.servicebus.windows.net:443/
+    eventHubNamespaceName:
+      type: String
+      value: ehn-dapr-mce123-trafficcontrol
+    iotHubName:
+      type: String
+      value: iothub-dapr-mce123
+    keyVaultName:
+      type: String
+      value: kv-dapr-mce123
+    logicAppAccessEndpoint:
+      type: String
+      value: https://prod-29.southcentralus.logic.azure.com:443/workflows/9bd179c8dd7049b8a152e5f2608f8efc
+    logicAppName:
+      type: String
+      value: logic-smtp-dapr-mce123
+    redisCacheName:
+      type: String
+      value: redis-dapr-mce123
+    serviceBusEndpoint:
+      type: String
+      value: https://sb-dapr-mce123.servicebus.windows.net:443/
+    serviceBusName:
+      type: String
+      value: sb-dapr-mce123
+    storageAccountContainerName:
+      type: String
+      value: trafficcontrol
+    storageAccountKey:
+      type: String
+      value: 7Ck76nP/5kFEhNx6C...V85L+0dFMFOA/xJLIvK25f2irUmVouPRbSGXKEzRQ==
+    storageAccountName:
+      type: String
+      value: sadaprmce123
+    ```
+
+    Copy these values into Notepad. You'll need them to configure your Dapr services.
+
+1.  Run the following command to fetch the AKS credentials for your cluster.
```

Comment:
Not going to request any changes here, but for vNext of this hack, would it be possible to automate steps 8-13 below via a script to remove the amount of steps the students have to do?

Reply:
Notepad is specific to Windows. You might want to say text editor to cover other OS's.